Assets/Scripts/PlayerCharacter.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerCharacter : MonoBehaviour {
+
+    [SerializeField]
+    private float accelerationForce = 5f;
+
+    [SerializeField]
+    private float jumpForce = 5f;
+
+    [SerializeField]
+    private float maxSpeed = 5f;
+
```

Comment:
If above variables are serialized fields, they don't need to be set in-code

Reply:
it's OK to initialize them (this gives default values in the editor), but I wouldn't set them elsewhere.